src/templates/index.html

```diff
@@ -53,13 +88,40 @@
 
     <div id="sidebar" class="leaflet-sidebar collapsed">
         <!-- Nav tabs -->
+        
         <div class="leaflet-sidebar-tabs">
             <ul role="tablist"> <!-- top aligned tabs -->
-                <li><a href="#home" role="tab"><i class="fa fa-info-circle"></i></a></li>
+                <li><a href="#uploadFile" role="tab"><i class="fa fa-upload"></i></a></li>
+                <li><a href="#home" role="tab"><i class="fa fa-info-circle"></i></a></li>           
             </ul>
         </div>
+
 <!-- Tab panes -->
+
 <div class="leaflet-sidebar-content">
+    <div class="leaflet-sidebar-pane" id="uploadFile">
+        <h1 class="leaflet-sidebar-header">
+            <div class="leaflet-sidebar-close"><i class="fa fa-caret-left"></i></div>
+        </h1>
+        <div class="leaflet-sidebar-content" style="padding-left: 20px;">
+            <h2>Input Excel File </h2>
+            <!--p>Please upload an Excel file adding relevant details 
+                that include Latitude and Longitude of the location, Hose Length(m),
+                Transport mode and Point Type.  </p-->
+            <p>The application requires the uploaded excel file, adhering to a specific data structure. 
+                Therefore, we recommend downloading our template <a href="https://drive.google.com/uc?export=download&id=12pwJ57A27z_MDwMeAm_Hl4W4nlI6a4Bz" download>(Click here)</a>
+                which includes all necessary inputs and data validation rules.
```

Comment:
Let's make a public github repo, or hugging face repo, make it with the DSSG google account as owner, then upload the file there and share that link, not your personal g drive link. 

I can provide the link, you just need to change and test.

Reply:
Update: try using this URL:

https://github.com/DSSGxMunich/static-files/raw/main/FileTemplate.xlsx